test/framework/framework.go

```diff
@@ -304,26 +304,18 @@ func (f *Framework) CreateOrUpdatePrometheusOperator(ctx context.Context, ns str
 		return nil, errors.Wrap(err, "wait for AlertmanagerConfig v1beta1 CRD")
 	}
 
-	// TODO(ArthurSens): The OperatorUpgrade tests won't pass because the operator v0.63.0 doesn't have the agent CRD.
-	// This check can be removed after the next release of the operator.
```

Comment:
Thanks and sorry for forgetting about this one 🤦

Reply:
no problem!